pages/_app.js

```diff
@@ -13,6 +14,19 @@ function MyApp({ Component, pageProps }) {
   const [fishLocation, setFishLocation] = useState("");
   const [markerPosition, setMarkerPosition] = useState([]);
 
+  const [fetchedData, setFetchedData] = useState([]);
```

Comment:
fetchedData is a non descriptive name. You should name your variables so they describe what kind of data they hold.